docs/corruption-detection-and-recovery.md

```diff
@@ -0,0 +1,62 @@
+# etcd Corruption Detection and Recovery
+
+Currently, in etcd 3.5.0-3.5.2, there is a low probability risk of data corruption across the etcd members [Upstream Issue](https://github.com/etcd-io/etcd/issues/13766), [Openshift Documentation](https://access.redhat.com/solutions/6849521). There is a mitigation coming to the Openshift Cluster etcd Operator that enables the etcd 'experimental-initial-corrupt-check' which should help detect corruption and stop the bad member from joining the cluster.
```

Comment:
minor, newline every 100ish characters. Doesn't have to be exact, but helps with comment association.

Reply:
ack, should be a little better.